vmm/sandbox/src/stratovirt/factory.rs

```diff
@@ -85,17 +85,25 @@ impl VMFactory for StratoVirtVMFactory {
         // create pcie.0 root bus object
         vm.pcie_root_bus = create_pcie_root_bus();
 
+        let mut transport = Transport::Pci;
```

Comment:
if the transport is strongly related to the machine type, maybe we can add a `transport()` in Machine to determine the transport by the machine type.

Reply:
fixed